Kconfig.nrf

```
@@ -51,7 +51,7 @@ config WARN_EXPERIMENTAL
 # Zephyr default is newlib and not newlib-nano for consistency between architectures.
 # To reduce FLASH footprint, newlib-nano is preferred default in NCS when newlib is selected.
 config NEWLIB_LIBC_NANO
-	default y
+	default y if !BOARD_MPS2
```

I think it's better that we only default to nano for newlib for nRF devices:
```suggestion
	default y if SOC_FAMILY_NORDIC_NRF
```

---

Fixed